content/src/service/validations/server.ts

```diff
@@ -39,8 +43,10 @@ const localChecks = async (entity: Entity, serviceCalls: ServiceCalls): Promise<
     return `This entity was already deployed. You can't redeploy it`
 
   /** Validate the deployment is not rate limited */
-  if (await serviceCalls.isEntityRateLimited(entity))
+  if (await serviceCalls.isEntityRateLimited(entity)) {
+    components.metrics.increment('dcl_content_rate_limited_deployments_total', { entity_type: entity.type })
```

Comment:
Do we have another metric to count the total number of deployments (rate limited or not), so we can't track the rate? If so, that's fine. If not, wouldn't it be nice to have a label that says if the deployment was rate limited or no?

Reply:
That way we can keep track of a the real load in order to adjust the limits, if necessary

Reply:
There are several different metrics referring to the number of deployments made (failing or not / from the sync or organic), and they should be only one with different tags. Anyway, that implies a refactor not only in code but also in Grafana dashboards, so I'd leave that to another issue.